### DISCUSSION TITLE
Looking for an open source tool to adjust screen brightness by change gamma

### DISCUSSION THREAD

**Discussion by ws2bhhh** [emoacht/Monitorian#641] [Q&A] [answered]
I apologize if I'm posting in the wrong place for this, I really need a software of this type. I just couldn't find the right community to post the question.

As the title suggests, my screen doesn't support DDC and the project developer won't add this feature to the project (which obviously makes sense considering the project usage haha).

Preferably an open source project, I'm using a suspicious software work that way, it asks for administrator rights and pops up a payment popup about once every 5 minutes, which is annoying.

**Comment by emoacht** [comment, marked as answer]
Unfortunately, this is not the right place for your question.
